content/config/completion.md

```diff
@@ -0,0 +1,106 @@
+---
+title: CLI completion
+description: Set up your shell to get autocomplete for Docker commands and flags
+keywords: cli, shell, fish, bash, zsh, completion, options
+---
+
+You can generate a shell completion script for the Docker CLI using the `docker
+completion` command. The completion script gives you word completion for
+commands, flags, and Docker objects (such as container and volume names) when
+you hit `<Tab>` as you type into your terminal.
+
+You can generate completion scripts for the following shells:
+
+- [Bash](#bash)
+- [Zsh](#zsh)
+- [fish](#fish)
+
+## Bash
+
+To Get Docker CLI completion with Bash, you first need to install the
+`bash-completion` package which contains a number of Bash functions for shell
+completion.
+
+```bash
+# Install using APT:
+sudo apt install bash-completion
+
+# Install using Homebrew (Bash version 4 or later):
+brew install bash-completion@2
+# Homebrew install for older versions of Bash:
+brew install bash-completion
+
+# With pacman:
+sudo pacman -S bash-completion
+```
+
+After installing `bash-completion`, source the script in your shell
+configuration file (in this example, `.bashrc`):
+
+```bash
+# On Linux:
+cat <<EOT >> ~/.bashrc
+if [ -f /etc/bash_completion ]; then
+    . /etc/bash_completion
+fi
+EOT
+
+# On macOS / with Homebrew:
+cat <<EOT >> ~/.bashrc
+if [ -f $(brew --prefix)/etc/bash_completion ]; then
+    . $(brew --prefix)/etc/bash_completion
+fi
+EOT
+```
```

Comment:
Sorry I should've been more specific here. This is shown after installing `bash-completion` with homebrew.

```suggestion
cat <<EOT >> ~/.bash_profile
[[ -r "$(brew --prefix)/etc/profile.d/bash_completion.sh" ]] && . "$(brew --prefix)/etc/profile.d/bash_completion.sh"
EOT
```

Reply:
https://formulae.brew.sh/formula/bash-completion@2#default